tools/README.md

```diff
@@ -0,0 +1,12 @@
+This folder contains scripts to build and deploy the sample applications to Bluemix.
+
+
+* #### watson-bluemix-cf  
+   This script deploys the sample application to Cloud Foundry on Bluemix. The script first pulls the *ibmjava:8-maven* image and
+   then uses this image to build the application. It then pushes the build application to Bluemix using CF Java Buildpack with IBM java.
```

Comment:
Dont add new lines, the entire description has to be one paragraph. Same for all paragraph's below.